samples/MvcSandbox/MvcSandbox.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Does this site work when started from the command line? I suspect you need a `<Content Update...>` because the Web SDK only sets `$(CopyToPublishDirectory)` for content it matches, not `$(CopyToOutputDirectory)`.